frontend/public/locales/en/public.json

```diff
@@ -1682,6 +1682,9 @@
   "ConsolePlugins": "ConsolePlugins",
   "Open Support Case with Red Hat": "Open Support Case with Red Hat",
   "Report Bug to Red Hat": "Report Bug to Red Hat",
+  "Non-parallel": "Non-parallel",
+  "Fixed Completion Count": "Fixed Completion Count",
```

Comment:
Should we change these to follow the new capitalization rules: "Fixed completion count?"

Reply:
Good point. So would it also be "Work queue" instead of "Work Queue"?

Reply:
Yes, if these are table columns, they should use sentence case.

Reply:
Changed the capitalization rules.